db/schema.rb

```diff
@@ -11,34 +11,38 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150924234313) do
+ActiveRecord::Schema.define(version: 20151015120755) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "actions", force: :cascade do |t|
-    t.integer  "person_id",    null: false
-    t.integer  "activity_id",  null: false
-    t.datetime "created_at",   null: false
-    t.datetime "updated_at",   null: false
+    t.integer  "person_id",       null: false
+    t.integer  "activity_id",     null: false
+    t.datetime "created_at",      null: false
+    t.datetime "updated_at",      null: false
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.